.golangci.yml

```diff
@@ -0,0 +1,69 @@
+run:
+  build-tags: 
+    - aws
+    - packet
+    - aks
+    - e2e
+    - baremetal
+    - disruptivee2e
+    - poste2e
+
+  modules-download-mode: vendor
```

Comment:
With this patch, it is no longer possible to build using different modules (not something we would do anyway, but pointing out reduced flexibility).